common/beerocks/bwl/dummy/mon_wlan_hal_dummy.cpp

```diff
@@ -150,6 +150,13 @@ bool mon_wlan_hal_dummy::sta_link_measurements_11k_request(const std::string &st
     return true;
 }
 
+bool mon_wlan_hal_dummy::channel_scan_trigger(int dwell_time_msec,
+                                              const std::vector<unsigned int> &channel_pool)
+{
+    LOG(TRACE) << __func__ << " - NOT IMPLEMENTED";
```

Comment:
nitpick: `__PRETTY_FUNCTION__` is generally more useful in C++ (https://gcc.gnu.org/onlinedocs/gcc/Function-Names.html)
Although non-standard.

Reply:
In my opinion its "pretty" only for  development stage.
Not sure that it will give any added value to a QA engineer while debugging.
Also its significantly increased log file size.

Reply:
It might be, depending on how many subclasses implement the method, and how many function overrides there are for the method (and templates, namespaces etc).
If there's more than one version of the function, I would want to know which one was triggered, even as a QA engineer while debugging. In fact, especially then.